<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>460</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="TE_inputFiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>415</width>
      <height>26</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Path to an Input File(-s)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="TE_outputFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>415</width>
      <height>26</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Path to the File(-s) Output Folder</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="TE_outputFileName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>365</width>
      <height>26</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter Output File Name</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CB_outputFileFormat">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>.webm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.mp4</string>
     </property>
    </item>
   </widget>
   <widget class="QToolButton" name="OD_inputFiles">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="OD_outputFolder">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="GEN_randomName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>440</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Random Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_beginConvert">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>440</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>440</width>
      <height>270</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>438</width>
       <height>268</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
